unyt/array.py

```diff
@@ -2214,15 +2214,16 @@ def uconcatenate(arrs, axis=0):
     >>> from unyt import cm
     >>> A = [1, 2, 3]*cm
     >>> B = [2, 3, 4]*cm
-    >>> uconcatenate((A, B))
+    >>> uconcatenate((A, B)) # doctest: +IGNORE_EXCEPTION_DETAIL
+    Traceback (most recent call last):
+    DeprecationWarning: ...
```

Comment:
Including the deprecation warning details in the doctest is gross but might be unavoidable. I can try poking at this too. I tell myself that running the doctests is worth the occasional uglyness that shows up.

Reply:
I couldn't find a more elegant solution yet, but I figure "keeping docstrings pretty in deprecated functions" might not be worth the effort (assuming it's even possible).
Happy to simplify this if you have an idea how to, however.